exonum_derive/src/lib.rs

```diff
@@ -27,32 +27,58 @@ mod tx_set;
 use proc_macro::TokenStream;
 use syn::{Attribute, Lit, Meta, MetaList, MetaNameValue, NestedMeta, Path};
 
-/// Exonum derive attribute names, used as `#[exonum( ATTRIBUTE_NAME = .. )]`
+/// Exonum derive attribute names, used as `#[exonum( ATTRIBUTE_NAME = .. or ATTRIBUTE_NAME )]`
 const CRATE_PATH_ATTRIBUTE: &str = "crate";
 const PB_CONVERT_ATTRIBUTE: &str = "pb";
+const SERDE_PB_CONVERT_ATTRIBUTE: &str = "serde_pb_convert";
 
 /// Derives `ProtobufConvert` trait.
+///
 /// Attributes:
-/// `#[exonum( pb = "path" )]`
-/// Required. `path` is name of the corresponding protobuf struct(generated from .proto file)
-/// `#[exonum( crate = "path" )]`
-/// Optional. `path` is prefix of the exonum crate(usually "crate" or "exonum")
+///
+/// * `#[exonum( pb = "path" )]`
+/// Required. `path` is the name of the corresponding protobuf generated struct.
+///
+/// * `#[exonum( crate = "path" )]`
+/// Optional. `path` is prefix of the `exonum` crate(usually "crate" or "exonum").
+///
+/// * `#[exonum( serde_pb_convert )]`
```

Comment:
Are we sure that we need this as an optional attribute? I mean, why make it opt-in so that some structs implement Serialize and Deserialize while others do not? Why not automatically implement these traits for all transaction structs without marking them with an attribute?

Reply:
Yes, I agree it can be done by default, but I would like to leave it as an option, because conventional serde can be required for some application.

It can be done by default(and option would be to disable it) but it would require to add feature `with-serde` to .toml file of every crate since only this way rust-protobuf structs derive serde for it's structs.

Reply:
But we seem to already litter the crates with `with-serde` features, just look at all the examples which you had to change. I wanted to comment on that first, but then I noticed that this feature is already present in many other places and let it past.

Is there some way to avoid that? I think I read somewhere that this is exactly why usage of features is discouraged unless absolutely necessary and that separate crates usually work better. Would having some _exonum-rust-protobuf_ crate with pre-enabled `with-serde` feature help?